code/modules/mob/mob_movement.dm

```diff
@@ -112,6 +112,10 @@
 		if(L.incorporeal_move)	//Move though walls
 			Process_Incorpmove(direct)
 			return 0
+	if(iscarbon(mob))
+		var/mob/living/carbon/C = mob
+		if(C.remote_view)

```

Comment:
Just kinda remove the remote_view and CameraMovement and move the CameraMovement code into /mob/camera/aiEye/remote/relaymove(), then replace ```user.remote_eye``` with ```src```